### PR TITLE
words: just words, nothing but words

### PR DESCRIPTION
**Pull request by catzkorn**
nothing in relation to fish at all.

### CODE REVIEW

File: words/tails.txt [hunk 1]
```diff
@@ -429,3 +429,114 @@ bumblebee
...
+danio
+betta
+bigeye
+bicolor
+bitterling
+bleak
+blenny
+tuna
+boga
+duck
```
Comment (DentonGentry) [inline words/tails.txt:469]: sus. fish imposter.

**Review by creachadair — APPROVED** [review]
I'm here for the fish.

File: words/scales.txt [hunk 2]
```diff
@@ -207,3 +207,114 @@ pyruvate
...
+paridae
+porgy
+rohu
+rudd
+skate
+squeaker
+tailor
+uaru
+vimba
+wahoo
```
Comment (DentonGentry) [inline words/scales.txt:319]: Aka "Ono"

**Review by DentonGentry — APPROVED** [review]

File: words/tails.txt (hunk 1 above)
Comment (catzkorn) [reply words/tails.txt:469]: [I take my job seriously ](https://en.m.wikipedia.org/wiki/Bombay_duck)

Comment (DentonGentry) [reply words/tails.txt:469]: Whoa...

**Pull request merged**